rclpy/rclpy/logging.py

```diff
@@ -43,22 +43,22 @@ def clear_config():
     initialize()
 
 
-def set_logger_level(name, level, detailed_error=False):
+def set_logger_level(name: str, level: int | LoggingSeverity, detailed_error: bool = False):
```

Comment:
Looks like CI is unhappy due to Windows being on Python 3.8. Mind using `Union[int, LoggingSeverity]` instead?

```
    def set_logger_level(name: str, level: int | LoggingSeverity, detailed_error: bool = False):
E   TypeError: unsupported operand type(s) for |: 'type' and 'EnumMeta'
```